docs/book/provider_implementations/building_running_and_testing.md

```diff
@@ -0,0 +1,58 @@
+# Building
+
+## Fixup Makefile to include common Cluster API CRDs
+
+Apply the following patch:
+
+```bash
+diff --git a/Makefile b/Makefile
+index ac12c7e..9b4f945 100644
+--- a/Makefile
++++ b/Makefile
+@@ -22,12 +22,14 @@ install: manifests
+ 
+ # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+ deploy: manifests
+-       kubectl apply -f config/crds
+-       kustomize build config/default | kubectl apply -f -
++       cat provider-components.yaml | kubectl apply -f -
+ 
+ # Generate manifests e.g. CRD, RBAC etc.
+ manifests:
+-       go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
++       go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd
```

Comment:
This probably isn't ideal. Ideally we'd be able to leverage more of the auto generation from kubebuilder. For the AWS implementation we disabled most of it for expedience as we were working on break-fix after the initial CRD migration.

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/567